template_content/playground/state/state_demo.py

```diff
@@ -0,0 +1,52 @@
+from algokit_utils import LogicError
```

Comment:
This file needs to be called `demo.py` to make the VS Code F5 stuff work

Reply:
just found that out! Thanks!